.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
@Gawan-0e278eed this file shouldn't be committed

Reply:
File removed